frontend/src/reducers/windowHandler.js

```diff
@@ -109,6 +110,7 @@ const initialModalState = {
  */
 export const initialState = {
   connectionError: false,
```

Comment:
As teo mentioned above  - can't we just have `connectionError` and have type as value ?